controllers/translateController.js

```diff
@@ -21,9 +24,22 @@ const translateController = async (text, sourceLanguage, nameOfTranslation, opti
 
     language.files.forEach(fileName => {
       const { file: languageJson, parsedPath } = getOrCreateJsonFile(basePath, fileName);
-      setDeepValue(languageJson, nameOfTranslation, result);
+      const existProperty = has(languageJson, nameOfTranslation); 
+      let userWantsOverwrite = 'no';
+      let wantsOverwrite = false;
```

Comment:
Maybe the language should be more imperative, like "userRequestOverwrite" and "shouldOverwrite"

Reply:
Thanks! Was hard to define those variables but these that you propose sounds better

Reply:
Thanks! added on https://github.com/victor-heliomar/i18n-translation-generator/pull/18/commits/6d0a760168cb1164753dd774055126754a5c1c41